src/blue-theme.scss

```diff
@@ -43,3 +43,9 @@ $primary: $democracy-blue;
 [class*="toast"] {
     margin-bottom: 0 !important;
 }
+
+// wraps text that has whitespace (handy for column header and labels)
+[class*="wrap"] {
+    white-space: normal;
+    height: auto;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5707004)

Reply:
I'm not sure why Codacy is complaining about this now.  I'm reasonably sure it's fixed. Let's wait and see if Codacy feels better in the morning.

Reply:
Add yet another newline maybe...

Reply:
Check whether the file is crlf or lf formatted, and try adding yet another line to be sure? :-P